source/features/code-highlight.js

```diff
@@ -0,0 +1,70 @@
+import {h} from 'dom-chef';
+import prism from 'prismjs';
+import 'prismjs/components/prism-jsx';
+import 'prismjs/components/prism-bash';
+import 'prismjs/components/prism-git';
+import 'prismjs/components/prism-typescript';
+import 'prismjs/components/prism-scss';
+
+const supportedLang = {
+	javascript: new Set(['js']),
+	jsx: new Set(['react', 'reactjsx']),
+	bash: new Set(['zsh', 'sh', 'shell']),
+	git: [],
+	typescript: [],
+	html: [],
+	css: [],
+	scss: []
+};
+
+function correctLanguage(lang) {
+	if (Object.keys(supportedLang).includes(lang)) {
+		return lang;
+	}
+	for (const key in supportedLang) {
+		if (supportedLang[key].has && supportedLang[key].has(lang)) {
+			return key;
+		}
+	}
+}
+
+function string2dom(domString) {
+	const dom = document.createElement('div');
+	dom.innerHTML = domString;
+	return dom;
+}
+
+export default function () {
+	const postsContent = $('.tweet-text');
+
+	postsContent.each((i, el) => {
+		const regex = /```(\w.*)([\s\S]+)```/g;
```

Comment:
The `.` should not be there.

Reply:
`regex` => `codeBlockRegex`

Reply:
Hi @sindresorhus,
I've searched in the PrismJS DOC but seems doesn't support the highlight for the regex,
I can try to do myself a basic colorization if there is a regex code block 🌈🙂

But I think this takes me some times

Reply:
@tomma5o I think you misunderstood my comments, because I'm not sure what you're talking about :) I'm only asking you to remove the dot and rename the variable.

Reply:
Ahhhh ok Sorry 🙃 
i understand you want a syntax higlight for regex